cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -210,11 +211,23 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command, _ []string) er
 func (cc *createClusterOptions) directoriesToMount(clusterSpec *cluster.Spec, cliConfig *config.CliConfig) []string {
 	dirs := cc.mountDirs()
 	fluxConfig := clusterSpec.FluxConfig
-	if fluxConfig == nil || fluxConfig.Spec.Git == nil {
-		return dirs
+	if fluxConfig != nil && fluxConfig.Spec.Git != nil {
+		dirs = append(dirs, filepath.Dir(cliConfig.GitPrivateKeyFile))
+		dirs = append(dirs, filepath.Dir(cliConfig.GitKnownHostsFile))
+	}
+
+	if clusterSpec.Config.Cluster.Spec.DatacenterRef.Kind == v1alpha1.CloudStackDatacenterKind {
+		env, found := os.LookupEnv("EKSA_CLOUDSTACK_HOST_PATHS_TO_MOUNT")
+		if found && len(env) > 0 {
+			mountDirs := strings.Split(env, ",")
+			for _, dir := range mountDirs {
+				dir = strings.TrimSpace(dir)
+				if _, err := os.Stat(dir); err == nil {
```

Comment:
Let's validate like we do here:

https://github.com/aws/eks-anywhere/blob/5860242bd1a58e4d00ede98a7ebe2cc9a0ce70aa/pkg/validations/createvalidations/preflightvalidations.go#L23

https://github.com/aws/eks-anywhere/blob/286b6571d98a206ea1148ea9e1a921d80674e16a/pkg/validations/upgradevalidations/preflightvalidations.go#L24